app/facades/quote_facade.rb

```diff
@@ -0,0 +1,12 @@
+class QuoteFacade
+  def self.quotes_by_keyword(keyword)
+    quotes_data = QuoteService.get_10_quotes_by_keyword(keyword)
+    quotes = {total_quotes: quotes_data[:totalCount], quotes: []}
+    if quotes[:total_quotes] > 0
+      quotes[:quotes] = quotes_data[:results].map do |quote_data|
+        Quote.new(quote_data)
+      end
+    end
+    quotes
+  end
```

Comment:
Great job here! You could even consider creating some type of super PORO that holds onto quotes and total_quotes. But I like this too since you're breaking it up in your controller and sending over exactly what your view needs.